### ISSUE TITLE
How to obtain checkpoint?

### ISSUE THREAD

**Issue by HLinChen** [Mxbonn/visualwakewords#4]
Thanks for the code! However, when I want to run the model, I can't find ckeckpoint files. How can I obtain them? Also, if I try to reproduce results in the paper, how to set hyperparameters, like learning rate, batch size, weight decay, epochs etc? Looking forward to receiving your reply!
